paasta_tools/cli/cmds/status.py

```diff
@@ -838,13 +838,28 @@ def print_flink_status(
         f" {status.overview['slots-available']}/{status.overview['slots-total']} slots available"
     )
 
+    # Avoid cutting job name. As opposed to default hardcoded value of 32, we will use max length of job name
+    max_job_name_length = max(
+        [len(job["name"].split(".", 2)[2]) for job in status.jobs]
```

Comment:
not blocking: `job["name"].split(".", 2)[2]` is repeated below, perhaps you could define an internal function or lambda to avoid drift

Reply:
Created function instead of lambda. 
Reason : "E731 do not assign a lambda expression, use a def"